src/commands/commerce/store/create.ts

```diff
@@ -453,4 +456,17 @@ export class StoreCreate extends SfdxCommand {
         // TODO check if index was created successfully, all i can do is assume it was
         await this.statusFileManager.setValue('indexCreated', true);
     }
+
+    private getBuyerProfileUsingTemplate(template: string): string {
```

Comment:
Since multiple files use the same logic, it might be worth considering refactoring this as a utility function so that this logic lives in one place instead of multiple places. It also helps clear up confusion about multiple `getBuyerProfileUsingTemplate` functions

Reply:
Yeah I will do that later when I make sure I need the same logic.